app/assets/stylesheets/main.css.scss

```diff
@@ -32,3 +32,7 @@ header {
 body {
   font-size: 16px;
 }
+
+.select2 {
+  width : 200px;
```

Comment:
for a css property that specific, rather then using it on the select2 class which would be a common class for implementing multi select through out ones application, you should add another class for such settings.

and try using % rather then going for specific 'N'px.